www/widgets/DemoExampleMaker.js

```diff
@@ -134,6 +147,18 @@ class DemoExampleMaker extends Widget {
         <textarea name="dem-url"></textarea>
       </div>
     `
+
+    this._text = new Netitor ({
+      ele: 'div[name="dem-s-text"]',
+      code: 'in this step...',
+      wrap: true,
+      language: 'html'
+    })
```

Comment:
the hinting menu (auto complete) is getting a little screwy, best to disable it. also, there's a weird bug style thing that happens: if u have ur editor setting set to any of the alternative themes && then u open the demo maker it screws w/netnet's CSS a bit, we can fix this by setting the theme for this to match whatever the current netnet editor theme is
```js
this._text = new Netitor ({
      ele: 'div[name="dem-s-text"]',
      code: 'in this step...',
      wrap: true,
      hint: false, // let's add this
      lint: false, // let's add this
      theme: NNE.theme, // let's add this
      language: 'html'
    })
```